client/src/components/ChatSideBanner/ChatSideBanner.tsx

```diff
@@ -29,9 +29,9 @@ const ChatSideBanner = ({ loggedInUser }: Props): JSX.Element => {
   return (
     <Grid className={classes.chatSideBanner}>
       <Box className={classes.userPanel}>
-        <AvatarDisplay loggedIn user={loggedInUser} />
+        {/* <AvatarDisplay loggedIn user={loggedInUser} /> */}
         <Typography className={classes.userText} variant="h5">
-          {loggedInUser.email}
+          {/* {loggedInUser.profile.firstName} */}
```

Comment:
Since you have created a Chat-Side Banner in the Sidebar, I think we can delete this component?